### DISCUSSION TITLE
[QUESTION] - How celery handles django orm connection?

### DISCUSSION THREAD

**Discussion by iamnotagentleman** [celery/celery#7013] [Q&A]
As far as i know django manage the ORM connections in request-response cycle, when i was try use django ORM with fastapi, the connections created from fastapi was never closed. Because it's can not be managed by django so i manage this connections manually, Then a question came to my mind. how celery do that?, because celery a separated process and have no clue about django request response cycle.

Thank you in advance for your answers.

note: sorry for feature request label i cant see any question label.

[a comment by an automated account is not shown]

**Comment by auvipy** [comment]
you can have a look here https://testdriven.io/blog/celery-database-transactions/ ffor basic understanding

**Comment by nitishxp** [comment]
Does this effect the active number of connection since I am seeing the number of connection is too high on my Cloud SQL exhausting my quota limit.
